src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/k8s-version2/troubleshooting.mdx

```diff
@@ -0,0 +1,294 @@
+---
+title: Kubernetes integration errors v2
+type: troubleshooting
+tags:
+    - Integrations
+    - Kubernetes integration
+    - Troubleshooting
+metaDescription: 'Some of the more common error messages found in the infrastructure agent logs for New Relic Kubernetes integration v2.'
+freshnessValidatedDate: 2024-07-22
+---
+
+Check out some solutions to the most common Kubernetes integration errors in version 2. These errors show up in the standard non-verbose infrastructure agent logs. If you need more detailed logs working with New Relic support, for example, see [Kubernetes logs](/docs/kubernetes-pixie/kubernetes-integration/troubleshooting/get-logs-version/).
```

Comment:
I would rework this intro. For example, instead of "check out some solutions", summarize what's changed from version 1 to version 2, and explain more why they would need to go to that Kubernetes logs doc you referenced (it's unclear what "if you need more detailed logs working with support" means).